schemas/2017-11-15-privatepreview/Microsoft.DataMigration.json

```diff
@@ -775,14 +775,6 @@
     "DataMigrationServiceProperties": {
       "type": "object",
       "properties": {
-        "publicKey": {
-          "type": "string",
-          "description": "The public key of the service, used to encrypt secrets sent to the service"
-        },
-        "virtualSubnetId": {
-          "type": "string",
-          "description": "The ID of the Microsoft.Network/virtualNetworks/subnets resource to which the service should be joined"
-        }
```

Comment:
removing these properties leaves an empty property bag. This can be a problem because an empty property bag can match any object.

Reply:
putting the pubkey back as placeholder